src/Pages/ProductList/Component/Category.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from 'react';
+import styled from "styled-components";
+
+class Category extends Component {
+  state={
+    activeTab : 0,
+  }
+
+  handleTab = (index) => {
+    this.setState({
+      activeTab: index
+    })
+  }
+
+  render() {
+    const arr = [
+      "ALL",
+      "ART PRINTS",
+      "BOOKS",
+      "MAGAZINE", 
+      "NOTECARDS", 
+      "SUBSCRIPTIONS",
+    ];
```

Comment:
이런 목업 데이터는 render 안에서 선언하면 render 될 때 마다 변수가 새로 선언됩니다!
컴포넌트 바깥으로 뺴주세요!
그리고 이런 변하지 않는 변수는 대문자_대문자로 변수명을 설정해줍니다! 컨벤션이라고 생각하시면 돼요!

```suggestion
    const CATEGORY_DATA = [
      "ALL",
      "ART PRINTS",
      "BOOKS",
      "MAGAZINE", 
      "NOTECARDS", 
      "SUBSCRIPTIONS",
    ];
```

Reply:
대문자로 수정하는건 컨벤션이군요!! 앞으로 변수 사용할때 유의하겠씁니다!